1.7hm.c

```diff
@@ -0,0 +1,47 @@
+#include <stdio.h>
+#define maxAvailableNumber 100000
+
+int main() 
+{
+    int numbers[maxAvailableNumber] = { 0 };
+    int maxNumber = 0;
+
+    printf("This program writes simple numbers ");

```

Comment:
```suggestion
    printf("This program writes prime numbers less or equal than N.\n");
    printf("N should be less than %d.\n", maxAvailableNumber);
    printf("Enter the N: ");
```
1) Так они на английском называются. И в остальных сообщениях также.
2) Более информативно.